src/plugins/workspace/public/utils.ts

```diff
@@ -539,6 +539,50 @@ export const fetchDataSourceConnections = async (
   }
 };
 
+export const updateFullFillRelatedConnections = (
+  openSearchConnections: DataSourceConnection[],
+  directQueryConnections: DataSourceConnection[]
+): DataSourceConnection[] => {
+  return fulfillRelatedConnections(openSearchConnections, directQueryConnections);
```

Comment:
Can't get the point of adding a `updateFullFillRelatedConnections` method, can we use `fulfillRelatedConnections` directly?

Reply:
Thanks for pointing the issue, this appears redundant, I will delete it.